pkg/debug/scripts/collect-info.sh

```diff
@@ -373,6 +373,7 @@ ln -s /persist/netdump      "$DIR/persist-netdump"
 ln -s /persist/kcrashes     "$DIR/persist-kcrashes"
 ln -s /run                  "$DIR/root-run"
 cp -r /sys/fs/cgroup/memory "$DIR/sys-fs-cgroup-memory" >/dev/null 2>&1
+ln -s /persist/SMART_details* "$DIR/"
```

Comment:
If files are missing you end up having this link:
```
ln -s tmp/SMART_details* tmp/tmp/
ls -la tmp/tmp/
'SMART_details*' -> 'tmp/SMART_details*'
```
Are those jsons always there?